orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -114,6 +118,9 @@ def __call__(self, file_paths, image_processed_callback=None):
         ConnectionError:
             If disconnected or connection with the server is lost
             during the embedding process.
+
+        Exception:
+            If cancelled attribute is set to True (default=False).
```

Comment:
If this is exposed/documented it should have a more specific type then just `Exception`.